harness/determined/deploy/aws/deployment_types/base.py

```diff
@@ -31,6 +31,7 @@
     constants.cloudformation.CPU_ENV_IMAGE,
     constants.cloudformation.GPU_ENV_IMAGE,
     constants.cloudformation.LOG_GROUP_PREFIX,
+    constants.cloudformation.RETAIN_LOG_GROUP,
```

Comment:
`RetainLogGroup` parameter is only in `simple`, `simple-rds`, and `govcloud` yamls. it's not in `{efs,fsx,lore,secure}.yaml`. so it's not exactly correct to add it into `common` list. either it needs to be added to all templates, or it should be in `template_parameter_keys` only for the types which support it...

Reply:
thanks!